### DISCUSSION TITLE
[Question] Distortion coefficient of cameras

### DISCUSSION THREAD

**Discussion by Entongsu** [isaac-sim/IsaacLab#1055] [Q&A]
Hi,

I wonder whether it is possible to obtain the coefficient of the camera. Thank you.

**Comment by Mayankm96** [comment]
Could you please elaborate?

**Comment by glvov-bdai** [comment]
To the best of my knowledge, the pinhole camera model does not include distortion, and as such, it is not possible to obtain the coefficents.
